api/base/settings/defaults.py

```diff
@@ -79,7 +79,7 @@
     # Needs to go before CommonMiddleware, so that transactions are always started,
     # even in the event of a redirect. CommonMiddleware may cause other middlewares'
     # process_request to be skipped, e.g. when a trailing slash is omitted
-    'api.base.middleware.DjangoGlobalMiddleware',
+    #'api.base.middleware.DjangoGlobalMiddleware',
```

Comment:
There were previously errors with something else- people were commenting out `FlaskRequestMiddleware` until the `DjangoGlobalsMiddleware` fix landed in #3389. (one of the symptoms of the old bug was weird exception behavior)

If the new `DjangoGlobalsMiddleware` is also having problems, then absolutely let me know and send a test case. But please *don't* commit workarounds to git without at least reporting the bug. :camel: